pkgs/development/ocaml-modules/getopt/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, ocaml, findlib, ocamlbuild }:
+
+stdenv.mkDerivation {
+  pname = "getopt";
+  version = "20120615";
+
+  src = builtins.fetchurl {
```

Comment:
What can I do about this? The newest tag on github seems to be from 2004.

Reply:
Maybe suggest the LIGO’s developers to use a library that is maintained…

Reply:
I did a fork that builds with dune, is that something we could use instead?

Reply:
That’s up to you. But I was thinking about something like `cmdliner`.

Reply:
I realize that I might have been less than clear in what I meant. I have this fork of the library: https://github.com/ulrikstrid/ocaml-getopt/
Would it be okay to use that as the nixpkg? Not using getopt is not an option for ligo currently